src/frontend/app/toolbar/import-file/import-file.component.scss

```diff
@@ -1,12 +1,18 @@
-:host {
-    background-color: yellow;
-    display: flex;
-    flex-direction: column;
-    align-items: center;
+.import-file__input{
+    display:none;
 }
 
-label, input {
-    flex: 1 1 auto;
+.import-file__label-container{
```

Comment:
For this button could you remove the label text and use an icon instead? I would suggest the upload_file from https://material.io/resources/icons/?icon=import_export&style=outline

Reply:
Done!